InscryptionCommunityPatch/ResourceManagers/ActThreeBonesDisplayer.cs

```diff
@@ -257,6 +257,9 @@ private static void AttachBonesTVScreen(ref ResourceDrone __instance)
             tvTube.radialDistortion = 0.6f;
             tvTube.reflexMagnetude = 0.2f;
 
+            // Move the behind-the-scenes render camera out of the way of P03's Face:
```

Comment:
The change log for the Bones Displayer is a little wacky; that's because I actually have a code auto-format extension installed now, and it fixed a bunch of little code formatting issues. The meaningful change is here